### PR TITLE
update command line references

### PR DESCRIPTION
**Pull request by ozlerhakan**
Signed-off-by: Hakan Ozler <hakan.ozler@kodcu.com>

### CODE REVIEW

**Comment by nathanleclaire** [comment]
@moxiegirl @londoncalling Any preference/opinion on this?

**Comment by londoncalling** [comment]
LGTM

File: docs/reference/ls.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ for all running machines:
 
 To list all machine names with their driver in a table format you can use:
 
-    $ docker-machine ls --format "table {{.Name}}: {{.DriverName}}"
+    $ docker-machine ls --format "table {{.Name}} {{.DriverName}}"
```
Comment (nathanleclaire) [inline docs/reference/ls.md]: FYI, I think the colon was intentional, but the change is still OK with me

**Comment by nathanleclaire** [comment]
LGTM

**Pull request merged**